gizmo.py

```diff
@@ -0,0 +1,47 @@
+import gizmo
+
+  
+def hello(a,country='Finland'):
+  print('Hello %s, how are things in %s?'%(a, country) )
+
+def spell(word):
+  spell_word=''
+  for i in word:
+    spell_word=spell_word+i+'.'
+  spell_word = spell_word[:-1]
+  print(spell_word)
+
+def relative_path(subj):
+  names=[]
+  for i in subj:
+      subject_identifier=i
+      a=f"./subjects/mock_recording_{subject_identifier}.rec"
+      names.append(a)
+  return names
+
+class Gizmo:
+    def __init__(self,na):
+        self.name=na
+    
+    def speak(self):
+        print(self.name)
+
+    def multiplication_table(zero_out_multiples=None):
+
+        a=np.array([[1,2,3,4,5,6,7,8,9,10,11,12]])
+        b=np.transpose(a)
+        c=a*b
```

Comment:
This is a perfectly valid way to do it. It is clear and get's the job done.

For the sake of education, here are some alternative ways to achieve the same result:

```python
a = np.arange(1, 13)
c = np.outer(a, a)
```
or:
```python
a=np.array([1,2,3,4,5,6,7,8,9,10,11,12])
c=a[:, np.newaxis] * b[np.newaxis, :]
```
(you can use `np.newaxis` to add new dimensions to an array on the fly, super useful!)